pkg/crypto/api.go

```diff
@@ -37,4 +39,10 @@ type Crypto interface {
 	// VerifyMAC determines if mac is a correct authentication code (MAC) for data
 	// using a matching MAC primitive in kh key handle and returns nil if so, otherwise it returns an error.
 	VerifyMAC(mac, data []byte, kh interface{}) error
+
+	// WrapKey will do ECDH key wrapping of cek using apu, apv and recipient public key found in kh.
```

Comment:
This comment is at the interface level. Are you saying the concrete implementation must do ECDH?

Reply:
I'll update the comment since default implementation is going to be ECDH, the interface doesn't need to enforce it.

Reply:
updated